docs/reference/schema.md

```diff
@@ -0,0 +1,539 @@
+---
+```
+
+## Hazard
+
+The hazard schema stores data about the intensity and occurrence probability of physical hazard phenomena such as floods, earthquakes, wildfires or others. The specific hazard process can be defined and measured with a specific intensity unit. For example, earthquake hazard may be represented as ground shaking, liquefaction or ground displacement.
+
+```{eval-rst}
+ .. mermaid::
+
+  classDiagram
+      Event set -- Event1
+      Event set -- Event2
+      Event set: Hazard type
+      Event set: Analytical method
+      class Event1{
+        Occurrence frequency
+        Time reference
+        Hazard trigger
+      }
+      class Event2{
+        Occurrence frequency
+        Time reference
+        Hazard trigger
+      }
+      Event1 -- Footprint1
+      Event1 -- Footprint2
+      Event2 -- Footprint3
+      Event2 -- Footprint4
+      class Footprint1{
+        Hazard process
+        Intensity measure
+        Uncertainty
+      }
+      class Footprint2{
+        Hazard process
+        Intensity measure
+        Uncertainty
+      }
+      class Footprint3{
+        Hazard process
+        Intensity measure
+        Uncertainty
+      }
+      class Footprint4{
+        Hazard process
+        Intensity measure
+        Uncertainty
+      }
+```
+
+The hazard schema specifies which type of analysis and data methodology that has generated the dataset. It supports either simulated probabilistic scenarios and empirical observations. If the dataset has been produced for a specific location, such a city, the name of the location can be included.
+
+When the scenario modelled refers to a specific period of time, this can be specified in terms of dates, period span and reference year. For example, an observed flood event that occurred from 1.10.2009 (time start) to 3.10.2009 (time end), spanning over 3 days (time span). When precise time collocation is unknown or not applicable, a general reference date such as "2009" is used to identify events (time year). This is also useful to specify future scenario, e.g. time year: 2050.
+
+When instead the hazard scenario is represented in probabilistic terms, the occurrence probability (frequency distribution) of hazard can be expressed in different ways. The most common way to communicate this is the "return period", expressed as the number of years after which a given hazard intensity could occur again: RP 100 indicates that that event has a probability of once in 100 years. This attribute can indicate individual layer frequency (RP100) or a range of frequencies for a collection of layers (RP10-100) The probability of occurrence is usually calculated on the basis of a reference period that provides observations: this period can be specified by start date, end date and time span. For example, an analysis of earthquake frequency based on seismic observations from 1934 (occurrence time start) to 2001 (occurrence time end), for a total count of 66 years (occurrence time span).
```

Comment:
This whole paragraph will need rewritten I think as the way occurence is modelled in the updated schema is very different

Reply:
Agreed, but, as mentioned in the issue description, I think updating the text is best left until the schema updates are done.